lib/LedgerSMB/Entity/Note.pm

```diff
@@ -8,7 +8,7 @@ employees, etc.
   @notes = LedgerSMB::Entity::Bank->list($entity_id, [$credit_id]);
   $note->add;
 
-=head1 DESCRIPTION
+=head1 DESCRIPTION 
```

Comment:
Really, I had not noticed.  Ha!

I have a perltidyrc that seems to fit the newer code formating rules.   Even if it is just what folk like, it 
seems like torture to pour a lot of reformats on top of all these refactorings.